docs/introduction.md

```diff
@@ -16,15 +16,15 @@ end-user can interact with the system.
 
 <p style="text-align:center"><img src="assets/layers.png"/></p>
 
-## Election flow
+## Form flow
 
-The following sequence diagram shows the entire flow of an election.
+The following sequence diagram shows the entire flow of an form.
 
-<p style="text-align:center"><img src="assets/election flow.png"/></p>
+<p style="text-align:center"><img src="assets/form flow.png"/></p>
```

Comment:
You can generate the new sequence diagram but copy/pasting the source bellow there: https://sequencediagram.org/.

Reply:
Sorry, I'm not sure to understand. Did you mean that I need to past the link in the source line 23 ?